charts/zookeeper/values.yaml

```diff
@@ -79,3 +83,7 @@ hooks:
 
 containers: []
 volumes: []
+#initContainers:
```

Comment:
Since the initContainer is really specialized for the volume permission effort (in the code we actually look for initContainers[0] explicitly, I think it would be clearer to bind then together here and have a section like:
```
volumePermissions:
  enabled: true
  initContainer:
    securityContext:
      runasUser: 0  # and lose the auto keyword.  We probably just need a value or no value (which means auto)
```

Reply:
Modified accordingly